### PR TITLE
Restore CODE_OF_CONDUCT.md from the wiki

### LINKED ISSUE #1: What happened to the Code of Conduct?
[The current Code of Conduct](https://github.com/privacyguides/.github/blob/f44b324ad36d3bc3719594c8254f9ff34a05f129/CODE_OF_CONDUCT.md) is very short, removed enforcement, our responsibilities etc. entirely. What happened to the [previous Code of Conduct](https://web.archive.org/web/20210824161038/https://wiki.privacytools.io/view/PrivacyTools:Code_of_Conduct)?

### PR DESCRIPTION
**Pull request by Mikaela**
Resolves: #1

### CODE REVIEW

**Comment by Mikaela** [comment]
It's in mediawiki syntax as I noticed it can be gotten out of Wayback Machine easily, I need to leave right now though, so I will return to it later and hopefully get it to markdown easily with pandoc with manual corrections where needed.

E.g. it still refers to Tools.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,7 +1,80 @@
...
+account, or acting as an appointed representative at an online or
+offline event. Representation of a project may be further defined and
+clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may
+be reported by contacting the *Internal Compliance Officer*. The report should
+include information on whether the details can be shared with other
+*Compliance Team* and *Core Team* Members and how much may be
```
Comment (Mikaela) [inline CODE_OF_CONDUCT.md:64]: These *emphatised* parts used to be links, but I have no idea where they were going to, so I just turned them like this as a note to fix them or discuss who these people are.

**Review by freddy-m — APPROVED** [review]
Happy for someone to merge this. cc @dngray

**Review by dngray — CHANGES_REQUESTED** [review]
Few things:

- What does this actually add over the original CoC? To me it t seems like it spells things out a bit more but that is all. I do believe it creates an opening for rule lawyering, rather than giving moderators discretion to remove/warn people of unwanted behavior.
- I do think we should wait to see @jonaharagon's thoughts on this before merging, as he decidedly didn't bring the PrivacyTools CoC forward
- I think we're too small for a specially designated "Internal Compliance Officer", what if the complaint is against the person wearing that hat
- I also think we're more than capable writing our own CoC and don't need the help of contributor-covenant.org to do that.

**Comment by jonaharagon** [comment]
Missed this, but I'm inclined to agree with @dngray, I think avoiding unnecessary complexities is an important part of our new direction, and I think the CoC as proposed here will encourage the "rule lawyering" mentioned, whereas the current CoC gives us more flexibility and discretion over community situations.

**Pull request closed without merging**